tests/e2e/tests/pilot/listener_conflict_test.go

```diff
@@ -219,25 +219,29 @@ func (i *pilotInfo) String() string {
 }
```

Comment:
This change is required for this PR. Otherwise the test wont work for the distroless variant. 
I fear, if we split this up, it will take one additional week (not effort) to get this merged into master.
The reason for this change is that `curl` is no longer installed in the docker image. Therefore, we need to use a different (testing) pod to run the `curl` command.